src/utils/multi_transport_connection.ts

```diff
@@ -0,0 +1,75 @@
+import * as t from "io-ts";
+
+import { NonEmptyString } from "italia-ts-commons/lib/strings";
+
+/**
+ * Describes a nodemailer transport connection
+ */
+export const MailTransportConnection = t.interface({
+  password: t.string,
+  transport: NonEmptyString,
+  username: t.string
+});
+
+export type MailTransportConnection = t.TypeOf<typeof MailTransportConnection>;
+
+/**
+ * An array of nodemailer transport connections
+ */
+export const MailMultiTransportConnections = t.readonlyArray(
+  MailTransportConnection
+);
+
+export type MailMultiTransportConnections = t.TypeOf<
+  typeof MailMultiTransportConnections
+>;
+
+/**
+ * Parses a multi email provider connection string.
+ *
+ * Multiple providers can be specified as following:
+ *
+ * transport:username:password[;transport:username:pasword][;transport:username:pasword]...
```

Comment:
```suggestion
 * transport:username:password[;transport:username:password][;transport:username:password]...
```

Reply:
what's 'transport' in this context ? the name of nodemailer transport ?

Reply:
yes